### PR TITLE
removed unused SwiftWebSocket9.0

### PR DESCRIPTION
**Pull request by or-else**
I'm sending it in a separate PR because I'm not sure I've removed it correctly. The code compiles though.

### CODE REVIEW

File: Tinodios/AppDelegate.swift [hunk 1]
```diff
@@ -126,7 +126,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     @discardableResult
     private func fetchDesc(for topicName: String) -> UIBackgroundFetchResult {
         let tinode = Cache.getTinode()
-        guard tinode.isConnectionAuthenticated || Utils.connectAndLoginSync() else {
+        guard tinode.isConnectionAuthenticated || Utils.connectAndLoginSync(inBackground: true) else {
             return .failed
```
Comment (or-else) [inline Tinodios/AppDelegate.swift:130]: This seems to be a Github bug. The `next` branch has identical code here:
https://github.com/tinode/ios/blob/next/Tinodios/AppDelegate.swift#L91

File: Tinodios/AppDelegate.swift [hunk 2]
```diff
@@ -126,7 +126,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     @discardableResult
     private func fetchDesc(for topicName: String) -> UIBackgroundFetchResult {
         let tinode = Cache.getTinode()
-        guard tinode.isConnectionAuthenticated || Utils.connectAndLoginSync() else {
+        guard tinode.isConnectionAuthenticated || Utils.connectAndLoginSync(inBackground: true) else {
```
Comment (or-else) [inline Tinodios/AppDelegate.swift:129]: This seems to be a Github bug. The next branch has identical code here:
https://github.com/tinode/ios/blob/next/Tinodios/AppDelegate.swift#L91

**Comment by or-else** [comment]
No. I just removed it in Xcode. There were two: ...10 and ...9. I deleted the 9.

Xcode was complaining about ambitious targets.

**Comment by aforge** [comment]
> No. I just removed it in Xcode. There were two: ...10 and ...9. I deleted the 9.
> 
> Xcode was complaining about ambitious targets.

Okay. Please run these commands to see if `pod` updates any files and merge the PR.

**Comment by or-else** [comment]
Sorry, saw your message only after merging.

**Pull request merged**